src/EventSubscriber/OgEventSubscriber.php

```diff
@@ -69,7 +69,11 @@ public function provideDefaultOgPermissions(PermissionEventInterface $event) {
    *   The default role event.
    */
   public function provideDefaultRoles(DefaultRoleEventInterface $event) {
-    $event->addRole(OgRoleInterface::ADMINISTRATOR, ['label' => 'Administrator']);
+    $event->addRole([
```

Comment:
Unless it is going to have a negative effect on your moral, I think it's better to have this change here. I'd prefer not to introduce a complexity where we know there's a simpler solution.